src/components/ui/Panel.jsx

```diff
@@ -101,21 +125,25 @@ class Panel extends React.Component {
   removeListeners() {
     document.removeEventListener('touchmove', this.move);
     document.removeEventListener('mousemove', this.move);
+    window.removeEventListener('resize', this.handleResize);
   }
 
   startResize = event => {
-    this.startHeight = this.panelDOMElement.offsetHeight;
+    const rect = this.panelDOMElement.getBoundingClientRect();
     this.startClientY = getEventClientY(event.nativeEvent);
+    this.startClientYOffset = this.startClientY - rect.top;
+    this.startHeight = window.innerHeight - rect.top;
 
     this.removeListeners();
```

Comment:
Yes, thank you I removed the `resize` event in willUnmount now. I don't see any reason why we don't remove the `move` listeners in stopResize instead of startResize ?

Reply:
> I don't see any reason why we don't remove the move listeners in stopResize instead of startResize ?

We actually do it in both :) (first line of `stopResize` method). But I think the one in start is useless.